pkg/util/kubernetes/apiserver/apiserver.go

```diff
@@ -108,9 +108,8 @@ func getKubeClient(timeout time.Duration) (kubernetes.Interface, error) {
 }
 
 func getInformerFactory() (informers.SharedInformerFactory, error) {
-	timeoutSeconds := time.Duration(config.Datadog.GetInt64("kubernetes_informers_restclient_timeout"))
```

Comment:
Should this be cleaned up from the config?

Reply:
yes